docs/operating-system/debugging.md

```diff
@@ -9,21 +9,27 @@ This section is not for end users. End users should use the [SSH add-on] to SSH
 
 [SSH add-on]: https://github.com/home-assistant/hassio-addons/tree/master/ssh
 
-## SSH access to the host
+## SSH access
+
+Home Assistant has two different types of ssh access.
+- End user SSH on port 22. This port is used by the [SSH add-on], it provides easy but limited SSH access to the system.
+- Host access SSH on port 22222. This method provides full access to the host with all privileges.
+
+### Enable SSH host access
 
 :::info
-SSH access through the [SSH add-on] (which will give you SSH access through port 22) will not provide you with all the necessary privileges, and you will be asked for a username and password when typing the 'login' command. You need to follow the steps below, which will setup a separate SSH access through port 22222 with all necessary privileges.
+A public/private key pair is required when setting up SSH host access, see [Generating SSH Keys](#generating-ssh-keys) for help.
 :::
 
-### Home Assistant Operating System
+Format a USB drive with FAT, ext4 or NTFS and name it `CONFIG` (case-sensitive). Create a file called `authorized_keys` (no extension) in the root of the USB drive, containing your public key. The file needs to be ANSI encoded and with Unix line ends (LF).
 
-Use a USB drive formatted with FAT, ext4, or NTFS and name it CONFIG (case sensitive). Create an `authorized_keys` file (no extension) containing your public key, and place it in the root of the USB drive. File needs to be ANSI encoded (not UTF-8) and must have Unix line ends (LF), not Windows (CR LF). See [Generating SSH Keys](#generating-ssh-keys) section below if you need help generating keys. Use the CLI (eg. SSH to the [SSH add-on] on port 22) and import the `authorized_keys` file with the `ha os import` command. You can now access your device as root over SSH on port 22222. Alternatively, the file will be imported from the USB when the Home Assistant OS device is rebooted.
+Next connect the USB drive to your Home Assistant OS device and import the public key. This can be either done via a CLI command over ssh on port 22 (using the [SSH add-on]) or via a reboot of the system (the file will be imported on reboot automatically). The CLI command when connected via SSH on port 22 is `ha os import`.
```

Comment:
The last sentence should be part of the story (and not dangling as a "forgotten" sentence at the end of the paragraph). Please note it doesn't require SSH access (nor does it have to be on port 22 specifically).

Reply:
Fixed